.travis.yml

```diff
@@ -188,6 +188,9 @@ matrix:
           env: SETUP_CMD='test' PIP_DEPENDENCIES='pyparsing requests'
                PIP_DEPENDENCIES_FLAGS='--log pip_install.log'
 
+        - os: linux
+          env: SETUP_CMD='test' CONDA_ENVIRONMENT='conda_environment.yml'
```

Comment:
Please also add another job with a dummy ``CONDA_DEPENDENCIES`` or ``PIP_DEPENDENCIES`` value as well, and then can use a custom ``TEST_CMD`` to make sure they are indeed not installed.

Reply:
Is there any reason to not allow additional dependencies to be installed on top of the environment?

Reply:
I don't see a reason to disallow it. I'd expect such behavior to be additive.

But I should probably add a test that the ones defined in the env actually do get installed.

Reply:
Actually it looks like you understood it, I just wrote the wrong thing and apparently changed my mind while doing this PR :)